index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HTML Homework</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <style>
+        body {
+            margin: 0;
+            color: #ffeeaa;
+            font-weight: 700;
+            line-height: 2;
+            font-size: calc(14px + (30 - 14) * ((100vw - 300px) / (1600 - 300)));
+        }
+
+        .grid-container {
+            display: grid;
+            grid-template-areas: 'l l n n n n n n n n n n' 'a a m m m m m m m m m m' 'f f f f f f f f f f f f';
+            grid-template-columns: 205px auto;
+            grid-template-rows: 60px auto 60px;
+            grid-gap: 5px;
+            background-color: #ffeeaa;
+            padding: 20px;
+            min-height: calc(100vh - 40px);
```

Comment:
instead of adding calc function, better set `box-sizing: border-box;` property to `.grid-container`. In this case padding will be considered a part of this element and you could set `min-height: 100vh;`